salsah/src/public/vendor/ckeditor-4.7.1-Editor/config.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+CKEDITOR.editorConfig = function( config ) {
```

Comment:
I do not really understand when `CKEDITOR.editorConfig` is called <https://docs.ckeditor.com/#!/api/CKEDITOR-method-editorConfig>